### DISCUSSION TITLE
[CodeStyle] Function names should correspond to the functionality of the function

### DISCUSSION THREAD

**Discussion by ViMaSter** [Studio-Lovelies/GG-JointJustice-Unity#164] [Ideas]
> Function names should correspond to the functionality of the function. Please call this out during PRs if it is not the case

---

This feels a bit too loose, but I'm also not sure how to phrase it more descriptively.  
Also I think the `Please call this out during PRs if it is not the case` suffix is not really needed. I would say that this applies to everything inside the style, no need to explicitly state it on this rule.

I also usually try to **start methods with a verb**, as they do things. (`GenerateX()`, `InstantiateY()`, `SaveZ()`)  
Maybe that's something we want to add?

**Comment by PhoebeMitchell** [comment]
Possibly something along the lines of "A method's functionality should be deducible from its name". Or maybe in plainer english: "you should be able to tell what a method does from just its name".
